ide/editor.autosave/i18n/omegat-ja/omegat/project_stats.txt

```diff
@@ -0,0 +1,41 @@
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+Estatística do projeto
+
+Total:
+	Segmentos	19
+	Palavras	185
+	Caracteres (sem espaços)	1242
+	Caracteres (incluindo espaços)	1398
+Únicos:
+	Segmentos	15
+	Palavras	174
+	Caracteres (sem espaços)	1158
+	Caracteres (incluindo espaços)	1307
+Segmentos únicos restantes:
+	Segmentos	2
+	Palavras	43
+	Caracteres (sem espaços)	329
+	Caracteres (incluindo espaços)	360
+
+Estatística do arquivo individual:
+
+Nome do arquivo	Total de palavras	Palavras restantes para traduzir	Total de caracteres (sem espaços)	Caracteres restantes (sem espaços)	Total de caracteres (incluindo espaços)	Caracteres restantes (incluindo espaços)
+javahelp\org\netbeans\modules\autosave\docs\autosave-about.html	69	1	439	13	501	13
+javahelp\org\netbeans\modules\autosave\docs\autosave-toc.xml	42	42	316	316	347	347
+src\org\netbeans\modules\autosave\Bundle.properties	74	0	487	0	550	0
```

Comment:
Again, an autogenerated file; better put a project-level .gitignore that excludes "project_stats.txt", or better, avoid depending on omegat and use internationalization bundles like other NetBeans modules.